common/constants.go

```diff
@@ -190,8 +190,6 @@ const (
 	DefaultOffset  = 0
 	DefaultLimit   = 20
 	CommaSeparator = ","
-	ValueYes       = "yes"
-	ValueNo        = "no"
 	ValueTrue      = "true"
 	ValueFalse     = "false"
```

Comment:
Do we really need these constants for `true and `false`??
```suggestion
	ValueTrue      = true
	ValueFalse     = false
```

Reply:
yes, we still need them for the query parameter definition

Reply:
they should still be in string data type, because the REST query parameter only allows string value

Reply:
Ok, can we then parse them to bool so that the `IssueGetCommandByName` func can use bool?

Reply:
Sure, `IssueGetCommandByName` client can use bool directly, and `ValueTrue`/`ValueFalse` will be used in the core-command internal implementation.